dydx/dydxPresenters/dydxPresenters/_v4/TakeProfitStopLoss/dydxTakeProfitStopLossViewPresenter.swift

```diff
@@ -295,6 +292,7 @@ private class dydxTakeProfitStopLossViewPresenter: HostedViewPresenter<dydxTakeP
             AbacusStateManager.shared.triggerOrders(input: $0, type: .stoplossusdcdiff)
         }
         viewModel.customAmountViewModel?.onEdited = {
+            print("mmm: custom amount edited \($0)")
```

Comment:
Delete?

Reply:
i'll do a clean up PR at the end of the project. There's many other print statements i need to get rid of. Omitted anyways 260a94a0ec2d0d65bc487dde945ade3feeff08e0